weave/environment.py

```diff
@@ -82,6 +82,19 @@ def weave_log_format(default: "logs.LogFormat") -> "logs.LogFormat":
     return LogFormat(os.getenv("WEAVE_LOG_FORMAT", default))
 
 
+def weave_link_prefix() -> str:
+    """When running in server we mount index under /weave"""
+    if os.getenv("GORILLA_ONPREM") == "true":
+        return "/weave"
+    return ""
+
+
+def analyics_disabled() -> bool:
+    if os.getenv("WEAVE_DISABLE_ANALYTICS") == "true":
```

Comment:
Nit: can simplify to `return os.getenv("WEAVE_DISABLE_ANALYTICS") == "true"` unless you're leaving the door open for more complex logic here.